### PR TITLE
replaces a string with a defined constant

### PR DESCRIPTION
**Pull request by zjcomeon**
**What type of PR is this?**
/kind bug

**What this PR does / why we need it**:
>replaces a string with a defined constant

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

File: cloud/pkg/cloudhub/handler/messagehandler.go [hunk 1]
```diff
@@ -142,7 +142,7 @@ func (mh *MessageHandle) HandleServer(container *mux.MessageContainer, writer mu
 			mh.MessageAcks.Delete(container.Message.Header.ParentID)
 		}
 		return
-	} else if container.Message.GetOperation() == "upload" && container.Message.GetGroup() == modules.UserGroup {
```
Comment (fisherxu) [inline cloud/pkg/cloudhub/handler/messagehandler.go:145]: Seems this's `upload`, not `update`.

Comment (zjcomeon) [reply cloud/pkg/cloudhub/handler/messagehandler.go:145]: I add a new constant

Comment (fisherxu) [reply cloud/pkg/cloudhub/handler/messagehandler.go:145]: Looks good, we also have sme hard code for `upload` in eventbus, could you please also fix them?

Comment (zjcomeon) [reply cloud/pkg/cloudhub/handler/messagehandler.go:145]: ok.I will modify it in the next pr

Comment (fisherxu) [reply cloud/pkg/cloudhub/handler/messagehandler.go:145]: Thanks, looks good, could you please squash the commits?

**Comment by fisherxu** [comment]
Please squash the commits, thanks.

File: cloud/pkg/cloudhub/handler/messagehandler.go (hunk 1 above)
Comment (zjcomeon) [reply cloud/pkg/cloudhub/handler/messagehandler.go:145]: Ok.Please check it

File: cloud/pkg/cloudhub/handler/messagehandler.go [hunk 2]
```diff
@@ -142,7 +142,7 @@ func (mh *MessageHandle) HandleServer(container *mux.MessageContainer, writer mu
...
+	} else if container.Message.GetOperation() == beehiveModel.UpdateOperation && container.Message.GetGroup() == modules.UserGroup {
```
Comment (fisherxu) [inline cloud/pkg/cloudhub/handler/messagehandler.go:145]:
```suggestion
	} else if container.Message.GetOperation() == beehiveModel.UploadOperation && container.Message.GetGroup() == modules.UserGroup {
```

Comment (fisherxu) [reply cloud/pkg/cloudhub/handler/messagehandler.go:145]: Should replace all to `upload`.

**Pull request closed without merging**